todo_app/app.py

```diff
@@ -8,4 +8,24 @@
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    items = session.get_items()
+    sortedItems = sorted(items, key=lambda item: item['status'], reverse=True)
```

Comment:
One last comment - the naming convention in Python is snake_case for variable and function names.

So `sorted_items`.